File: 1186A_Contest.cpp
#include <iostream>
using namespace std;

/*
Entrada:
 - Tres enteros: x a b (x = número de participantes, a = bolígrafos, b = cuadernos)
Salida:
 - "YES" si a >= x y b >= x, sino "NO"
Ejemplo:
Entrada:
3 3 4
Salida:
YES
*/

int main () {
    int x, a, b;
    cin >> x >> a >> b;  

    if (((a - x) >= 0) && ((b - x) >= 0)) {
        cout << "YES" << endl;
    } else {
        cout << "NO" << endl;
    }
    
    return 0;
}

File: 1186A_Contest.py
"""
Entrada:
 - Tres enteros: x a b (x = número de participantes, a = bolígrafos, b = cuadernos)
Salida:
 - "Yes" si a >= x y b >= x (puede dar al menos 1 de cada a cada participante), sino "No"
Ejemplo:
Entrada:
3 3 4
Salida:
Yes
"""

x, a, b = map(int,input().split())
if a - x >= 0 and b - x >= 0:
    print("Yes")
else:
    print("No")